test/unit/lib/much-rails/call_method_callbacks_tests.rb

```diff
@@ -47,58 +44,55 @@ def initialize
 
     should "know its attributes" do
       # before_callbacks
-      assert_that(
-        subject.much_rails_call_callbacks_config.before_callbacks.size
-      ).equals(0)
-
       subject.before_call(&proc1)
-      assert_that(
-        subject.much_rails_call_callbacks_config.before_callbacks.size
-      ).equals(1)
-      assert_that(subject.much_rails_call_callbacks_config.before_callbacks)
-        .equals([proc1])
-
-      subject.prepend_before_call(&proc2)
-      assert_that(
-        subject.much_rails_call_callbacks_config.before_callbacks.size
-      ).equals(2)
-      assert_that(subject.much_rails_call_callbacks_config.before_callbacks)
-        .equals([proc2, proc1])
 
-      # after_callbacks
-      assert_that(subject.much_rails_call_callbacks_config.after_callbacks.size)
-        .equals(0)
+      assert_that(-> { subject.before_call(&proc2) })
+        .changes(
+          "subject.much_rails_call_callbacks_config.before_callbacks.dup",
+          from: [proc1],
+          to: [proc1, proc2]
+        )
```

Comment:
Prefer using the block slot with `assert_that { ... }.changes(...)`:

```suggestion
      assert_that { subject.before_call(&proc2) }
        .changes(
          "subject.much_rails_call_callbacks_config.before_callbacks.dup",
          from: [proc1],
          to: [proc1, proc2]
        )
```
... same for the other usages.